src/ResourceManager/Sql/Commands.Sql/ChangeLog.md

```diff
@@ -19,11 +19,14 @@
 -->
 ## Current Release
 * Fixed issue with default resource groups not being set.
-* Deprecated unsupported LongTermRetention features:
-	* Get-AzureRmSqlServerBackupLongTermRetentionVault cmdlet
-	* Set-AzureRmSqlServerBackupLongTermRetentionVault cmdlet
-	* Get-AzureRmSqlDatabaseBackupLongTermRetentionPolicy -Current parameter
-	* Set-AzureRmSqlDatabaseBackupLongTermRetentionPolicy -State parameter
+* Breaking changes for LongTermRetention cmdlets:
+	* Begin deprecation process:
+		* Get-AzureRmSqlServerBackupLongTermRetentionVault cmdlet
+		* Set-AzureRmSqlServerBackupLongTermRetentionVault cmdlet
+		* Get-AzureRmSqlDatabaseBackupLongTermRetentionPolicy -Current parameter
+		* Set-AzureRmSqlDatabaseBackupLongTermRetentionPolicy -State parameter
+	* Breaking changes:
```

Comment:
This structure makes it seem like this is actually a breaking change rather than a deprecation message.  I think it would make more sense to have one header "Deprecated cmdlets and parameters"

Reply:
Sure